packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -23,7 +23,7 @@ import { usePagination } from './hooks/features/pagination/usePagination';
 import { useRows } from './hooks/features/rows/useRows';
 import { useSorting } from './hooks/features/sorting/useSorting';
 import { useApiRef } from './hooks/features/useApiRef';
-import { useColumnReorder } from './hooks/features/useColumnReorder';
+import { useColumnReorder } from './hooks/features/column-reorder';
```

Comment:
file name convention

```suggestion
import { useColumnReorder } from './hooks/features/columnReorder';
```

Reply:
Ok, I was wondering what should we use as some of them are with dashes and some camel case.

Reply:
When you have a doubt, check the most popular pattern used in https://github.com/mui-org/material-ui.

Reply:
Actually, I have been wondering about doing "field trips", having community members work on x for a week and the other way around, as a way to share knowledge, better understand each team constraints, etc.

Reply:
I would like that, yes it would be nice also to share the knowadge of the deliverables.